.travis.yml

```diff
@@ -112,6 +113,8 @@ script:
   - python -c 'import numpy as np; print("relaxed strides checking:", np.ones((10,1),order="C").flags.f_contiguous)'
   # Make sure that relaxed strides checking is actually in effect; otherwise fail loudly
   - if [ "$NPY_RELAXED_STRIDES_CHECKING" == "1" ]; then python -c'import numpy as np; assert np.ones((10,1),order="C").flags.f_contiguous'; fi
+  # Test that mpmath actually uses gmpy2
+  - python -c 'import mpmath.libmp; assert mpmath.libmp.BACKEND == "gmpy"'
```

Comment:
On Thu, Jul 28, 2016 at 08:39:13AM -0700, Evgeni Burovski wrote:
>  > +  # Test that mpmath actually uses gmpy2
>  > +  - python -c 'import mpmath.libmp; assert mpmath.libmp.BACKEND == "gmpy"'
> 
>    Not sure if this is too strict: test suite may well pass without gmpy2.

Suppose, one day gmpy2 build will be broken by a different reason.  How your
developers will catch such a problem?  If this test will be not "too
strict" - they will have to read whole build logs _every time_, instead
of just looking on the build status.

At the end, you should ask yourself - is this behaviour worth
efforts to support it?  I.e. build speedup is significant for you.
If not - just a warning would be better, of course)

(Another doable in principle option - open a github issue, if
this will be broken.  Looks too complex...)